src/main/java/com/denizenscript/clientizen/tags/ClientTagBase.java

```diff
@@ -12,6 +13,9 @@
 import com.denizenscript.denizencore.scripts.commands.core.AdjustCommand;
 import com.denizenscript.denizencore.tags.PseudoObjectTagBase;
 import com.denizenscript.denizencore.tags.TagManager;
+import net.fabricmc.loader.api.FabricLoader;
+import net.fabricmc.loader.impl.FabricLoaderImpl;
+import net.fabricmc.loader.impl.metadata.AbstractModMetadata;
```

Comment:
You should NEVER depend on impl stuff - it's ALWAYS subject to change, including the packages. This will also most likely break with Quilt immediately.

Reply:
Replaced it with just hard-coding the constants for now